src/reducer/chat.ts

```diff
@@ -52,5 +52,14 @@ export const chatReducer = (
         ...state,
         listConversation: newListConversation,
       };
+    case ChatActionKind.DELETE_CONVERSATION:
```

Comment:
I think this related to listConversation, so we should use action like as DELETE_LIST_CONVERSATION or UPDATE_LIST_CONVERSATION

Reply:
I think this name is ok, if you want it to be clearer, how about `DELETE_CONVERSATION_IN_LIST`?

Reply:
yes, it's good to me